actions/class.RdfController.php

```diff
@@ -1386,4 +1483,12 @@ private function getActionAccessControl(): ActionAccessControl
     {
         return $this->getServiceLocator()->get(ActionAccessControl::SERVICE_ID);
     }
+
+    private function isElasticSearchEnabled(): bool
```

Comment:
Please, lets avoid use `ElasticSearch` word here, cause it is storage related.

```suggestion
    private function isAdvancedSearchEnabled(): bool
```